requirements/requirements.txt

```diff
@@ -12,7 +12,7 @@ git+https://github.com/fedspendingtransparency/django-mock-queries#egg=django-mo
 django-queryset-csv==1.0.1
 django-simple-history==1.8.2
 django-spaghetti-and-meatballs==0.2.2
-Django==1.11.18
+Django==1.11.20
```

Comment:
Is this related to the sys.float_info.max usage later in this PR?  The release notes I found (https://docs.djangoproject.com/en/2.1/releases/1.11.19/) aren't completely clear. Just curious about the purpose of this version bump.

Reply:
No, they are unrelated